src/api/google-drive-api.js

```diff
@@ -214,3 +215,15 @@ function findNotesByName(name = '') {
     });
   });
 }
+
+function findNoteById(lastViewedNoteId) {
```

Comment:
Why is it a function of `lastViewedNoteId`? :)

This is for any note, so I believe it should be `function findNoteById(noteId)`

Reply:
Done!